repository/a.json

```diff
@@ -1746,6 +1746,17 @@
 			]
 		},
 		{
+			"name": "AutoItScript",
+			"details": "https://github.com/AutoIt/SublimeAutoItScript",
+			"releases": [
+				{
+					"sublime_text": "*",
+					"platforms": ["windows"],
+					"branch": "master"
```

Comment:
We no longer support branch-based releases for new packages. Instead, change this to `"tags": true` and create a [semver](http://semver.org) tag on the repo each time you want a new version to be released.